### DISCUSSION TITLE
如何用MVVM实现TreeView 的SelectItem绑定到右侧，显示其SelectedItem属性

### DISCUSSION THREAD

**Discussion by Neal2gx** [BYJRK/DotNet-Discussions#285] [WPF 开发]
![12](https://github.com/BYJRK/DotNet-Discussions/assets/26769093/36b1aef6-376f-4e3b-ab81-38c6693da12a)
如何用MVVM实现TreeView 的SelectItem绑定到右侧，显示其SelectedItem属性；
想实现：
![12](https://github.com/BYJRK/DotNet-Discussions/assets/26769093/0062ddb8-72b7-4ebb-a621-6b5a0ac46ada)
点击左侧Tree ViewItem，右边会显示选中的Item属性。并可以修改它

**Comment by arcanix601** [comment]
【从ASP.NET及EntityFramework出发探讨MVVM中的Model究竟是什么-哔哩哔哩】 https://b23.tv/7bj3dgF
这期视频里有讲

**Reply by BYJRK** [reply]
这期视频主要想说的是 MVVM 里面的 Model，以及通常会采用的 Dto 或 Model Wrapper 之类的方式。可能跟这个问题的关系并不大

**Comment by BYJRK** [comment]
最简单的方式是，右侧的 `DataContext` 可以直接绑定 `TreeView` 的 `SelectedItem` 属性

**Reply by Neal2gx** [reply]
好嘞，谢谢。

**Reply by hcyos** [reply]
能详细说明吗
